pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -22,9 +22,12 @@
 , graphviz
 , fmt
 , python39
```

Comment:
Should use python310 ideally, which is the default python version for NixOS 23.05.